cli/src/base.ts

```diff
@@ -43,12 +43,12 @@ export abstract class BaseCommand extends Command {
   searchPlaces = [`.${moduleName}rc`, `.${moduleName}rc.json`, 'package.json'];
 
   static databaseURLFlag = Flags.string({
-    name: 'databaseurl',
+    char: 'u',
     description: 'URL of the database in the format https://{workspace}.xata.sh/db/{database}'
   });
 
   static branchFlag = Flags.string({
-    name: 'branch',
+    char: 'b',
```

Comment:
Sorry if this is a silly question, but is this shorthand used? The help page doesn't show it:

<img width="960" alt="image" src="https://user-images.githubusercontent.com/9947422/176150418-19deae00-9c78-43e7-9293-cbaa6a2dcf93.png">

Reply:
Fixed here https://github.com/xataio/client-ts/pull/367/commits/116ea09970c6e51ddebf122deb64a152305cafe4